ircd/s_serv.c

```diff
@@ -142,6 +143,7 @@ init_builtin_capabs(void)
 	CLICAP_CAP_NOTIFY = capability_put(cli_capindex, "cap-notify", NULL);
 	CLICAP_CHGHOST = capability_put(cli_capindex, "chghost", &high_priority);
 	CLICAP_ECHO_MESSAGE = capability_put(cli_capindex, "echo-message", NULL);
+	CLICAP_EXTENDED_MONITOR = capability_put(cli_capindex, "draft/extended-monitor", &high_priority);
```

Comment:
Why does this need high priority? It's a draft cap, and anybody supporting it should support 3.2-style CAP LS (which doesn't need priority)

Reply:
Makes sense :+1: